### DISCUSSION TITLE
AdGuard Quick Fixes filter

### DISCUSSION THREAD

**Discussion by piquark6046** [uBlockOrigin/uBlock-issues#3356] [General]
https://github.com/AdguardTeam/AdguardFilters/pull/187308

**Comment by garry-ut99** [comment]
Unrelated to uBO, as it's for MV3 only and uBO already has it's own Quick fixes list, hence why to post here, it should rather go to uBOL repo or to uAssets repo. Also uBOL already has its dedicated list: https://github.com/uBlockOrigin/uAssets/blob/master/filters/ubol-filters.txt, not sure how "quick" it is though.

**Reply by gorhill** [reply]
If I understand correctly AdGuard MV3 will support dynamic update of filters between extension updates.

This is not the case for uBOL, so there is no need for such list.

**Reply by garry-ut99** [reply]
> Title: AdGuard Quick Fixes filter

Duplicate of:
- https://github.com/uBlockOrigin/uBOL-home/issues/108
(also discussed in https://github.com/uBlockOrigin/uAssets/issues/23732)

**Comment by krystian3w** [comment]
It remains to be seen whether it will be added to filterlists.com and how often it will fit in the reports above line "too many lists" and how much it will disrupt the operation of uBo MV2 to consider blocking it in badlists.txt then.

https://github.com/AdguardTeam/FiltersRegistry/pull/993/files#diff-c4b2d989b441fa1ddb800519b95077b3083763ce2562bcccd709fd44a26064c0R10
